doc/nrf/releases_and_maturity/releases/release-notes-2.9.0.rst

```diff
@@ -205,6 +205,8 @@ Bluetooth LE
   This was introduced in |NCS| 2.7.0 as experimental.
   The :ref:`bt_scanning_while_connecting` sample showcases how you can use this feature to reduce the time to establish connections to many devices.
 
+* Fixed an issue where Bluetooth applications built with the ``nordic-bt-rpc`` snippet (in the :ref:`bt_rpc` configuration) did not work on the nRF54H20 devices due to incorrect memory mapping.
```

Comment:
The link seems to fail the doc build. Do you mean to link to Bluetooth Low Energy Remote Procedure Call sample? in that case the link would be ble_rpc_host

Reply:
Yeah, its ble_rpc as this is the changelog entry for PR https://github.com/nrfconnect/sdk-nrf/pull/19292/files. 
Also, this entry should be mentioned in the v2.9.0-nRF54H20-1 Release Notes.